.github/workflows/test-font-render.yml

```
@@ -82,6 +82,7 @@ jobs:
 
   test-windows:
     needs: build
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)